plugins/source/mysql/client/sync.go

```diff
@@ -97,10 +124,6 @@ func (c *Client) resourceFromValues(tableName string, values []any) (*schema.Res
 	table := c.Tables.Get(tableName)
 	resource := schema.NewResourceData(table, nil, values)
 	for i, col := range table.Columns {
-		// If the value points to an empty byte array array we set it to nil
-		if v, ok := values[i].(**[]byte); ok && *v == nil {
```

Comment:
This was a workaround and no longer needed since the scalar UUID supports `**[]byte` due to https://github.com/cloudquery/plugin-sdk/pull/922